plugins/out_pgsql/pgsql.h

```diff
@@ -23,8 +23,10 @@
 #include <fluent-bit/flb_output.h>
 #include <fluent-bit/flb_time.h>
 #include <fluent-bit/flb_output_plugin.h>
-
+#ifdef linux
 #include <libpq-fe.h>
+#endif
+#include "pgsql_loadlib.h"
```

Comment:
this should contain an #ifdef FLB_SYSTEM_WINDOWS since it special case for win32 not other platforms